builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -0,0 +1,1062 @@
+              events.sendCompletedEvent('CREATE_ASSET', null, utilModule.generateAssetMap("aws", arn, "cloudwatch_event", config), environment_logical_id);
+            }
+          }
+
+          def envBucketKey = "${env_key}${configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX}"
+          echoServerlessFile()
+          def deployOutput = servelessDeploy(environment_logical_id, envBucketKey)
+
+// Shoving deploy output lines into array
+          def deployLines = deployOutput.split('\n')
+          def resourceArns = []
+          def afterStackOutputs = false
+// Extracting all the lines that follow 'Stack Outputs'. These lines are supposed to contain arns of resources created
+          for(def line: deployLines) {
+            if(afterStackOutputs) resourceArns.add(line)
+            if(line.contains('Stack Outputs')) afterStackOutputs = true // Here we are only interested in lines that follow 'Stack Outputs' marker
+          }
+
+          for(def name2ArnString: resourceArns) {
+            def name2ArnPair = name2ArnString.split(': ') // Here we rely on the space after the ': ' so it is split properly without affecting the arn itself that is full of colons but never has a space after
+            if(name2ArnPair.size() > 1) {
+              def arn = name2ArnPair[1].trim()
+              if(arn.startsWith('arn')) {
+                 def arnAsArray = arn.split(':') // Here we splitting the arn itself in hope to obtain type of the resource. Should be the third element: arn:aws:dynamodb:us-east-1:102707241671:table/jazzUsersTable53
+                 if(arnAsArray.size() > 2) { // Making sure that the third element exists
+                   def artifactType = arnAsArray[2]
+                   if(artifactType == 'kinesis') artifactType = 'kinesis_stream' // 'kinesis_stream' needs to be instaead of simple 'kinesis' that did not get persisted :-( by itself
+                   events.sendCompletedEvent('CREATE_ASSET',
+                                             null,
+                                             utilModule.generateAssetMap("aws",
+                                                                         arn,
+                                                                         artifactType,
+                                                                         config),
+                                             environment_logical_id)
+                   echo "Sending type $artifactType with arn $arn"
+
+                }
+              }
+            }
+          }
+
+
+          // reset Credentials
+          resetCredentials(credsId)
+          if (domain != "jazz") {
+            createSubscriptionFilters(stackName, configLoader.AWS.REGION, roleId);
+          }
+
+          if (domain && domain == "jazz") {
+            serviceConfigdata.setLogStreamPermission(config)
+            serviceConfigdata.setKinesisStream(config)
+          }
+          def svc_response = echoServiceInfo(environment_logical_id)
```

Comment:
This is N/A for sls_apps. As an app can have multiple functions, we can't determine a service endpoint as such. You can remove this.

Reply:
Addressing with commit: 'Removing serviceConfigdata.setLogStreamPermission(config)'

Reply:
Resolving with https://github.com/tmobile/jazz/pull/829/commits/37489156452ce4711302c37d6de8cc9081874313